markdown.dtx

```diff
@@ -15994,6 +16192,137 @@ following text:
 %
 % \begin{markdown}
 
+#### Tex Math Renderers
+The \mdef{markdownRendererInlineMath} and \mdef{markdownRendererDisplayMath} macros 
+represent inline and display \TeX{} math. 
+Both macros receive a single argument that corresponds to the tex math content.
+These macros will only be produced, when the \Opt{texMathDollars}
+option is enabled.
+
+% \end{markdown}
+%
+% \iffalse
+
+##### Plain \TeX{} Example {.unnumbered}
+
+Using a text editor, create a text document named `document.tex` with the
+following content:
+``` tex
+\input markdown
+\def\markdownOptionTexMathDollars{true}
+\def\markdownRendererInlineMath#1{$\underline{#1}$}
+\def\markdownRendererDisplayMath#1{$$\textstyle#1$$}
+\markdownBegin
+$E=mc^2$
+
+$$\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx$$
+\markdownEnd
+\bye
+```````
+Next, invoke LuaTeX from the terminal:
+``` sh
+luatex document.tex
+``````
+A PDF document named `document.pdf` should be produced and contain the
+following text:
+
+> $E=mc^2$
+> 
+> $$\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx$$
```

Comment:
In examples, I tend to show the output format in the user manual as well. Since this is especially difficult to do with math, I suggest showing just a LaTeX example with only the `displayMath` renderer redefined and adding `\quad(1)` before `$$` in the example output, so that the HTML user manual looks similar to the `equation` environment with its equation numbering:

![CodeCogsEqn](https://user-images.githubusercontent.com/603082/220460198-b6e7cd68-a351-49d4-b557-4f9f8a5fc926.png)